Server/mods/deathmatch/logic/CPedSync.cpp

```diff
@@ -86,25 +90,20 @@ void CPedSync::UpdatePed(CPed* pPed)
     // This ped got a syncer?
     if (pSyncer)
     {
-        // He isn't close enough to the ped and in the right dimension?
-        if ((!IsPointNearPoint3D(pSyncer->GetPosition(), pPed->GetPosition(), (float)g_TickRateSettings.iPedSyncerDistance)) ||
-            (pPed->GetDimension() != pSyncer->GetDimension()))
-        {
-            // Stop him from syncing it
-            StopSync(pPed);
+        // Is he close enough, and in the right dimension?
+        if (IsPointNearPoint3D(pSyncer->GetPosition(), pPed->GetPosition(), (float)g_TickRateSettings.iPedSyncerDistance))
+            if (pPed->GetDimension() == pSyncer->GetDimension())
+                return;
 
-            if (!pPed->IsBeingDeleted())
-            {
-                // Find a new syncer for it
-                FindSyncer(pPed);
-            }
-        }
-    }
-    else
-    {
-        // Try to find a syncer for it
-        FindSyncer(pPed);
+        // Stop him from syncing it
+        StopSync(pPed);
     }
+
+    if (pPed->IsBeingDeleted())
+        return;
```

Comment:
I think blocks without curly braces should be generally avoided.

Reply:
This is absolutely fine. Please see our [style guide](https://github.com/multitheftauto/mtasa-blue/wiki/Style-Guide) for more information.